components/openindiana/ca-certificates/manifests/sample-manifest.p5m

```diff
@@ -313,7 +312,7 @@ link path=etc/openssl/certs/8867006a.0 \
     target=../../certs/CA/GeoTrust_Universal_CA_2.pem
 link path=etc/openssl/certs/8cb5ee0f.0 \
     target=../../certs/CA/Amazon_Root_CA_3.pem
-link path=etc/openssl/certs/8d86cdd1.0 \
+link path=etc/openssl/certs/8d86cdd$(IPS_COMPONENT_VERSION) \
```

Comment:
ooops...

Reply:
Ugly, but I guess does not break anything?